docs/checks/internal/checks.yaml

```diff
@@ -123,44 +124,49 @@ checks:
     risk: High
     tags: supply-chain, security, source-code, code-reviews
     short: Determines if the default and release branches are protected with GitHub's branch protection settings.
-    description: >-
-      [Branch protection](https://docs.github.com/en/github/administering-a-repository/defining-the-mergeability-of-pull-requests/about-protected-branches)
-      can enable various rules to enforce certain workflows for
-      branches, such as preventing rewriting of public history
-      (e.g., a *force push*),
-      requiring review before acceptance into a main branch,
-      or passing certain status checks before acceptance into a main branch.
-
-      Branch protection can reduce the risk of unintentional or malicious
-      code from entering the "main" branch.
-      Branch protection rules that prevent rewriting of public history
-      (e.g., preventing *force push* of public branches) prevent history
-      from changing without external notice.
-      Branch protection rules that require status checks ensure that
-      at least those checks are met before a change is accepted.
-      Branch protection rules that require at least one other reviewer
-      reviews greatly reduces the risk that a compromise of a contributor's
-      account will lead to injection of malicious code. Review also increases
-      the likelihood that an unintentional vulnerability in a contribution
-      will be detected and fixed before the change is accepted.
-      A low score is therefore considered `High` risk.
-
-      Note, however, that requiring reviews for every proposed change
-      is impractical for many projects, since many projects simply don't
-      have that many active participants. For more discussion, see
-      [Code Reviews](#code-reviews).
-
-      In some cases these rules will need to be suspended.
-      For example, if a past commit includes illegal content
-      (such as child pornography), it may be impractical to hide the commit
-      and in such cases the history may need to be rewritten.
-
-      This check determines if the default and release branches are
-      protected with GitHub's branch protection settings.
-      The check only works when the token has [Admin
-      access](https://github.community/t/enable-branch-protection-get-api-without-admin/14197)
-      to the repository. This check determines if the default and release branches are
-      protected.
+    description: |
```

Comment:
maybe add a note about https://github.com/ossf/scorecard/issues/1106#issuecomment-936591864 somewhere? Just note that these settings in the link are not accessible without admin token. Or in the main readme.. or both. I don't know which is best.